src/components/Comments.jsx

```diff
@@ -0,0 +1,27 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+import { getAllCommentsByArticleId } from "../utils/api"
+
+function Comments ({article_id, singleArticle}){
+   const [commentsById, setCommentsById] = useState([])
+useEffect(()=>{
+    getAllCommentsByArticleId(article_id)
+   .then(({data})=>{
+    const commentsArr = data.comments
+    setCommentsById(commentsArr)
+   })
+},[])
```

Comment:
As this part also requires api call, better have isLoading state here as well. And don't forget to put article_id into the dependencies array of the useEffect